example/app/data/DemoDataSource/plugins/attribute-selector/blueprints/AttributeSelectorTabsBlueprint.json

```diff
@@ -3,6 +3,11 @@
   "name": "AttributeSelectorTabsBlueprint",
   "extends": ["CORE:NamedEntity"],
   "attributes": [
+    {
+      "attributeType": "string",
+      "type": "dmss://system/SIMOS/BlueprintAttribute",
+      "name": "type"
+    },
```

Comment:
Same as above, the attribute "type" should be included in CORE:NamedEntity, so I don't think it is necessary to add the "type" attribute to AttributeSelectorTabsBlueprint.

Could you double check this?